src/ceph-daemon/ceph-daemon

```diff
@@ -1252,6 +1267,7 @@ def command_run():
 
 def command_shell():
     # type: () -> int
+    infer_fsid()
```

Comment:
Having this all over the place seems wrong to me, though. can we call `infer_fsid` one place only?

Reply:
I did that deliberately so that we infer only in cases where it is not dangerous to do so. 'deploy' is used only by mgr/ssh, so we don't do it there, and we *definitely* don't want to for rm-cluster or rm-daemon.

Reply:
Maybe it's cleaner todo this as a python decorator ? 

https://github.com/mgfritch/ceph/commit/da6fa2feeccb975946239b39dcae58cd83a1c134